some integration with clients testing broke

today's build-package on ubuntu errored out https://github.com/datalad/git-annex/actions/runs/6938554232/job/18874466539
![image](https://github.com/datalad/git-annex/assets/39889/532d6182-c6fa-47ab-acae-1b85c898bdfb)

```
+ git ls-remote --heads origin 'build-*'
+ cut -f2
+ cut -d/ -f3-
+ git ls-remote --heads origin 'result-*'
+ cut -f2
+ cut -d/ -f3-
+ yq -r 'keys | .[]' ../clients/clients.yaml
+ read clientid
Error: open ../clients/clients.yaml: no such file or directory
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build-macos.yaml
+++ b/.github/workflows/build-macos.yaml
@@ -60,10 +60,10 @@ jobs:
         run: |
           # The goal here is for $BUILD_COMMIT to be the HEAD (necessary for
           # git-annex's version detection to use the correct git commit) with
-          # the .github/ and patches/ trees from master — or whatever ref is
-          # being used as the workflow source — also available.
+          # the .github/, clients/, and patches/ trees from master — or
+          # whatever ref is being used as the workflow source — also available.
           git checkout "$BUILD_COMMIT"
-          git checkout "$GITHUB_SHA" -- .github patches
+          git checkout "$GITHUB_SHA" -- .github clients patches
 
       - name: Get build version
         id: build-version

--- a/.github/workflows/build-ubuntu.yaml
+++ b/.github/workflows/build-ubuntu.yaml
@@ -79,10 +79,10 @@ jobs:
         run: |
           # The goal here is for $BUILD_COMMIT to be the HEAD (necessary for
           # git-annex's version detection to use the correct git commit) with
-          # the .github/ and patches/ trees from master — or whatever ref is
-          # being used as the workflow source — also available.
+          # the .github/, clients/, and patches/ trees from master — or
+          # whatever ref is being used as the workflow source — also available.
           git checkout "$BUILD_COMMIT"
-          git checkout "$GITHUB_SHA" -- .github patches
+          git checkout "$GITHUB_SHA" -- .github clients patches
 
       - name: Get build version
         id: build-version

--- a/.github/workflows/build-windows.yaml
+++ b/.github/workflows/build-windows.yaml
@@ -66,14 +66,14 @@ jobs:
         run: |
           # The goal here is for $BUILD_COMMIT to be the HEAD (necessary for
           # git-annex's version detection to use the correct git commit) with
-          # the .github/ and patches/ trees from master — or whatever ref is
-          # being used as the workflow source — also available.
+          # the .github/, clients/, and patches/ trees from master — or
+          # whatever ref is being used as the workflow source — also available.
           git reset --soft "$BUILD_COMMIT"
           # Avoid checking out unnecessary files with paths that are invalid on
           # Windows.
           git ls-tree --name-only HEAD | grep -v '^doc$' | xargs git checkout HEAD
           git checkout HEAD doc/license ':(glob)doc/*.mdwn' ':(glob)doc/logo*'
-          git checkout "$GITHUB_SHA" -- .github patches
+          git checkout "$GITHUB_SHA" -- .github clients patches
 
       - name: Get build version
         id: build-version

--- a/.github/workflows/template/build-{{ostype}}.yaml.j2
+++ b/.github/workflows/template/build-{{ostype}}.yaml.j2
@@ -96,8 +96,8 @@ jobs:
         run: |
           # The goal here is for $BUILD_COMMIT to be the HEAD (necessary for
           # git-annex's version detection to use the correct git commit) with
-          # the .github/ and patches/ trees from master — or whatever ref is
-          # being used as the workflow source — also available.
+          # the .github/, clients/, and patches/ trees from master — or
+          # whatever ref is being used as the workflow source — also available.
           {% if ostype == "windows" %}
           git reset --soft "$BUILD_COMMIT"
           # Avoid checking out unnecessary files with paths that are invalid on
@@ -107,7 +107,7 @@ jobs:
           {% else %}
           git checkout "$BUILD_COMMIT"
           {% endif %}
-          git checkout "$GITHUB_SHA" -- .github patches
+          git checkout "$GITHUB_SHA" -- .github clients patches
 
       - name: Get build version
         id: build-version

